cylc/flow/task_events_mgr.py

```diff
@@ -819,17 +838,21 @@ def _process_message_check(
             self.EVENT_SUBMIT_FAILED, f'{FAIL_MESSAGE_PREFIX}ERR'
         }:
             severity = DEBUG
-        LOG.log(severity, f"[{itask}] {flag}{message}{timestamp}")
+
+        if not quiet:
+            LOG.log(severity, f"[{itask}] {flag}{message}{timestamp}")
```

Comment:
Can move the severity calculation just above inside this `if not quiet` block

Reply:
Done locally - will push